File: MODUL 6/Python/PRAK604-2310817220011-ReginaSilvaMaharatini.py
asli = input()
palsu = input()
kode = len(asli)
pesan = len(palsu)
a = 0
b = 0
if len(asli) != len(palsu):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range(len(asli)):
        if asli[i] == palsu[i]:
            print("*", end='')
            a+=1
        else:
            print("#", end='') 
            b+=1
    print("* = {}" .format(a))
    print("# = {}" .format(b))
    if a >= b:
        print("Pesan Asli")
    elif a < b:
        print("Pesan Palsu")

File: MODUL 6/C/PRAK604-2310817220011-ReginaSilvaMaharatini.c
#include<stdio.h>
#include<string.h>
int main(){
     char asli[100];
     char palsu[100];
     scanf("%[^\n]%*c", &asli);
     scanf(" %[^\n]%*c", &palsu);
     int kode, pesan, n, a=0, b=0;
    kode = strlen(asli);
    pesan = strlen(palsu);
    if(kode != pesan){
          printf("Panjang kalimat berbeda, pesan palsu");
          }else{
          for(n=0; n<kode; n++){
                if(asli[n]==palsu[n]){
                    printf("*");
                    a++;} 
                else{
                    printf("#");
                    b++;}}
          printf("\n* = %d", a);
          printf("\n# = %d", b);          
     if(a>=b){
          printf("\nPesan Asli");}
     else if(a<b){
          printf("\n Pesan Palsu");}}    
return 0;}